back/src/lib/middlewares.go

```diff
@@ -16,7 +19,15 @@ func (airneis *airneisFiberApp) setupLogger() fiber.Router {
 }
 
 func (airneis *airneisFiberApp) setupCORS() fiber.Router {
-	return airneis.app.Use(cors.New())
+	return airneis.app.Use(cors.New(
+		cors.Config{
+			AllowCredentials: true,
+			AllowOriginsFunc: func(origin string) bool {
+				log.Infof("Origin: %s", origin)
```

Comment:
Is this log really necessary ?

Reply:
We log for clarity and to know clearly what is the origin of handled request.

Reply:
<img src="https://media2.giphy.com/media/5DQdk5oZzNgGc/giphy.gif"/>